internal/human/marshal.go

```diff
@@ -45,14 +45,17 @@ func Marshal(data interface{}, opt *MarshalOpt) (string, error) {
 
 	rType := rValue.Type()
 
+	// safely get the marshalerFunc
+	marshalerFunc, _ := getMarshalerFunc(rType)
```

Comment:
why do we work on `marshalerFunc` ? 

is this necessary to fix the problem of parallel tests ? 

shouldn't this be in another PR ?

Reply:
Changing the map to a sync map means changing the way we access the map, thus the getter :)